gateway/api.go

```diff
@@ -3268,3 +3271,21 @@ func updateOASServers(spec *APISpec, conf config.Config, apiDef *apidef.APIDefin
 	newAPIURL := getAPIURL(*apiDef, conf)
 	oasObj.UpdateServers(newAPIURL, oldAPIURL)
 }
+
+// GetSessionLifetime defines the TTL of the key
+func GetSessionLifetime(sess *user.SessionState, gw *Gateway) int64 {
```

Comment:
@sredxny we already have a `session.Lifetime()`. Don't you think this would cause confusion?

Reply:
yes, will rename it